### PR TITLE
[dss/schemas] Add schemas to docker image #908

### PR DESCRIPTION
**Pull request by barroco**
This PR addresses #908 by including db migration files to the dss docker image.

### CODE REVIEW

File: build/deploy/schema-manager.libsonnet
Comment (BenjaminPelletier) [inline build/deploy/schema-manager.libsonnet:4]: I think we can go ahead and remove these mounts directly -- if someone wants to deploy (e.g.) v0.5.0, my expectation would be that they would use the deployment tools in v0.5.0 of the repository, which would mount the schemas.  I don't think we need to support the use of v0.5.0 deployment tools to deploy v0.6.0 of the DSS image, which is what I think this deprecation notice + retaining the mounts is accomplishing.

Comment (barroco) [reply build/deploy/schema-manager.libsonnet]: Sounds good. done.

**Review by BenjaminPelletier — APPROVED** [review]
Looks good assuming you've done a trial deployment to verify (we don't have good automated tests for the deployment architecture yet).

Even though this PR is approved, let's wait to merge until after #905 as it has been pre-empted by other PRs already a few times now.

**Comment by barroco** [comment]
> Looks good assuming you've done a trial deployment to verify (we don't have good automated tests for the deployment architecture yet).
I generated the tanka manifest and reviewed it which looked ok. I will deploy it after #905 merge for complete validation.

**Comment by BenjaminPelletier** [comment]
#905 merged; I'll await validation via test deployment before merging this PR, but otherwise I think it's ready to go.

**Review by BenjaminPelletier — CHANGES_REQUESTED** [review]
Note that this PR is now behind the merge of #905 and should have master merged into it before a test deployment before merging.

**Review by BenjaminPelletier — APPROVED** [review]
Tested to deploy and probe properly on Google Cloud

**Pull request merged**
